package.json

```diff
@@ -55,7 +55,7 @@
 		"@types/glob": "^7.1.3",
 		"@types/mocha": "^9.1.1",
 		"@types/node": "^18.0.0",
-		"@types/vscode": "^1.55.0",
+		"@types/vscode": "^1.60.2",
```

Comment:
Does yarn.lock need to be updated? It still references 1.55 as the minimum and 1.71 as the actual version.